nx-dev/feature-doc-viewer/src/lib/related-documents-section.tsx

```diff
@@ -0,0 +1,139 @@
+import { RelatedDocumentsCategory } from '@nx/nx-dev/models-document';
+
+export function RelatedDocumentsSection({
+  relatedCategories,
+}: {
+  relatedCategories: RelatedDocumentsCategory[];
+}) {
+  return (
+    <div className="grid  lg:grid-cols-3 sm:grid-cols-2 grid-cols-1 gap-3 items-stretch">
+      {relatedCategories.length > 0 &&
+        relatedCategories.map((category) => (
+          <CategoryBox key={category.id} category={category} />
+        ))}
+    </div>
+  );
+}
+
+const iconMap: { [key: string]: JSX.Element } = {
+  concepts: (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      strokeWidth="1.5"
+      stroke="currentColor"
+      className="w-6 h-6 mr-2"
+    >
+      <path
+        strokeLinecap="round"
+        strokeLinejoin="round"
+        d="M21 7.5l-2.25-1.313M21 7.5v2.25m0-2.25l-2.25 1.313M3 7.5l2.25-1.313M3 7.5l2.25 1.313M3 7.5v2.25m9 3l2.25-1.313M12 12.75l-2.25-1.313M12 12.75V15m0 6.75l2.25-1.313M12 21.75V19.5m0 2.25l-2.25-1.313m0-16.875L12 2.25l2.25 1.313M21 14.25v2.25l-2.25 1.313m-13.5 0L3 16.5v-2.25"
+      />
+    </svg>
```

Comment:
You do not have to copy and paste the SVG from HeroIcons, use it directly from the library https://github.com/nrwl/nx/blob/ba35bf19c3c39d8b65faffd8be8fc35cc3b529d1/nx-dev/feature-ai/src/lib/error-message.tsx#L20-L22

Reply:
Also, add `aria-hidden="true"`.